README.md

```diff
@@ -26,13 +26,29 @@ Daedalus - Cryptocurrency Wallet
    ```
    and then add the following lines:
    ```
+   build-users-group = nixbld
+
+   max-jobs = auto
+   cores = 0                                         # the builder will use all available CPU cores
+   sandbox = false
+
+   require-sigs = true
+   trusted-users = root
+   allowed-users = *
+
    substituters = https://hydra.iohk.io https://cache.nixos.org/
    trusted-substituters =
    trusted-public-keys = hydra.iohk.io:f/Ea+s+dFdN+3Y/G+FDgSq+a5NEWhJGzdjvKNGv0/EQ= cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
-   max-jobs = 2  # run at most two builds at once
-   cores = 0     # the builder will use all available CPU cores
-   extra-sandbox-paths = /System/Library/Frameworks
+   extra-sandbox-paths = /System/Library/Frameworks /System/Library/PrivateFrameworks /usr/lib
+
+   # If you are using MacOS, please uncomment one of the following configs, based on your chip:
+
+   # extra-platforms = x86_64-darwin                 # Mac with Intel chip
```

Comment:
I dont think you need this. By default it should have `system = x86_64-darwin`; and there are not extra-platforms.

The whole extra-platforms logic is only needed on M1's as it technically allows to run two different architectures.
- x86_64-darwin via Rosetta2 and
- aarch64-darwin natively.

Reply:
You are right. I've updated the example to include only specific `system` for M1 to enforce Rosetta2.